configs/components/ruby-selinux.rb

```diff
@@ -1,3 +1,139 @@
+# Component release information:
+#   https://github.com/SELinuxProject/selinux/releases
+#####
+component 'ruby-selinux' do |pkg, settings, platform|
+    # We download tarballs because system development packages (e.g.
+  # libselinux-devel) don't necessarily include Swig interface files (*.i files)
+  # We select the minimum version available in the platform repos.
+  case platform.name
+  when /^(el-7|amazon-2|redhatfips-7)-/
+    pkg.version "2.0.94"
+    pkg.sha256sum 'b8312852306650e9720de5a20fe7560d935d3c90ffedca1cac25bf3f283d8a36'
+    pkg.url 'https://raw.githubusercontent.com/wiki/SELinuxProject/selinux/files/releases/20100525/devel/libselinux-2.0.94.tar.gz'
+  when /^(el-8|redhatfips-8)-/
+    pkg.version '2.9'
+    pkg.sha256sum '1bccc8873e449587d9a2b2cf253de9b89a8291b9fbc7c59393ca9e5f5f4d2693'
+    pkg.apply_patch 'resources/patches/ruby-selinux/selinux-29-function.patch'
+    pkg.url 'https://github.com/SELinuxProject/selinux/releases/download/20190315/libselinux-2.9.tar.gz'
+  when /^debian-11-/
+    pkg.version '3.1'
+    pkg.sha256sum 'ea5dcbb4d859e3f999c26a13c630da2f16dff9462e3cc8cb7b458ac157d112e7'
+    pkg.url 'https://github.com/SELinuxProject/selinux/releases/download/20200710/libselinux-3.1.tar.gz'
+    pkg.build_requires 'python3-distutils' if platform.is_deb?
+  when /^(el-9|redhatfips-9|ubuntu-22.04)-/
+    pkg.version '3.3'
+    pkg.sha256sum 'acfdee27633d2496508c28727c3d41d3748076f66d42fccde2e6b9f3463a7057'
+    pkg.build_requires 'python3-distutils' if platform.is_deb?
+    pkg.url "https://github.com/SELinuxProject/selinux/releases/download/#{pkg.get_version}/libselinux-#{pkg.get_version}.tar.gz"
+  when /^(amazon-2023|debian-12)-/
+    pkg.version '3.4'
+    pkg.sha256sum '77c294a927e6795c2e98f74b5c3adde9c8839690e9255b767c5fca6acff9b779'
+    pkg.build_requires 'python3-distutils' if platform.is_deb?
+    pkg.url "https://github.com/SELinuxProject/selinux/releases/download/#{pkg.get_version}/libselinux-#{pkg.get_version}.tar.gz"
+  when /^(sles-15|ubuntu-24.04)-/
+    pkg.version '3.5'
+    pkg.sha256sum '9a3a3705ac13a2ccca2de6d652b6356fead10f36fb33115c185c5ccdf29eec19'
+    pkg.build_requires 'python3-setuptools'
+    pkg.url "https://github.com/SELinuxProject/selinux/releases/download/#{pkg.get_version}/libselinux-#{pkg.get_version}.tar.gz"
+  when /^(fedora-41|ubuntu-25.04)-/
+    pkg.version '3.7'
+    pkg.sha256sum 'ea03f42d13a4f95757997dba8cf0b26321fac5d2f164418b4cc856a92d2b17bd'
+    pkg.build_requires 'python3-setuptools'
+    pkg.url "https://github.com/SELinuxProject/selinux/releases/download/#{pkg.get_version}/libselinux-#{pkg.get_version}.tar.gz"
+  when /^(el-10|redhatfips-10|fedora-42)-/
+    pkg.version '3.8'
+    pkg.sha256sum '0c3756bca047c9270281d7c4dcdecd000b72e38a183c930661eba9690839b541'
+    pkg.build_requires 'python3-setuptools'
+    pkg.url "https://github.com/SELinuxProject/selinux/releases/download/#{pkg.get_version}/libselinux-#{pkg.get_version}.tar.gz"
+  when /^(debian-13|sles-16)-/
```

Comment:
reading through https://software.opensuse.org/package/libselinux, opensuse 16 ships 3.9. Now I don't know how that relates to SLES 16 :(